istioctl/pkg/xds/client.go

```diff
@@ -100,6 +100,7 @@ func DialOptions(opts clioptions.CentralControlPlaneOptions,
 		return mcpDialOptions(ctx, opts.GCPProject, k8sCreds)
 	}
 	return []grpc.DialOption{
+		// #nosec G402
```

Comment:
I think we want `nolint: gosec` probably. But the important part is adding context for *why* we are opting out, otherwise it just encourages mindlessly skipping the lint